packages/components/src/FormField/FormFieldTypes.ts

```diff
@@ -18,6 +20,16 @@ export interface FieldActionsRef {
   setValue(value: string | number): void;
 }
 
+export interface Affix {
+  readonly label?: string;
+  readonly icon?: IconNames;
+}
+
+export interface Suffix extends Affix {
+  readonly icon: IconNames;
+  onClick?(): void;
+}
```

Comment:
^ I think every instance of onClick should require ariaLabel, since the suffix label isn't naming the action, it's a postfix for the input value (we might be saying the same thing, just want to make sure)

Reply:
yep! agreed and I believe we're saying the same thing 😂

Reply:
👍  resolved